content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md

```diff
@@ -621,21 +678,54 @@ Like before, all previous files in the `/etc/config/` directory will be deleted.
+This applies in the case where an optionally referenced ConfigMap comes into
+existence after a pod has started.
+
+Kubelet checks whether the mounted ConfigMap is fresh on every periodic sync. However,
+it uses its local TTL-based cache for getting the current value of the ConfigMap. As a
+result, the total delay from the moment when the ConfigMap is updated to the moment
+when new keys are projected to the pod can be as long as kubelet sync period (1
+minute by default) + TTL of ConfigMaps cache (1 minute by default) in kubelet. You
+can trigger an immediate refresh by updating one of the pod's annotations.
+
+{{< note >}}
+A container using a ConfigMap as a [subPath](/docs/concepts/storage/volumes/#using-subpath) volume will not receive ConfigMap updates.
+{{< /note >}}
 
 <!-- discussion -->
 
 ## Understanding ConfigMaps and Pods
 
-The ConfigMap API resource stores configuration data as key-value pairs. The data can be consumed in pods or provide the configurations for system components such as controllers. ConfigMap is similar to [Secrets](/docs/concepts/configuration/secret/), but provides a means of working with strings that don't contain sensitive information. Users and system components alike can store configuration data in ConfigMap.
+The ConfigMap API resource stores configuration data as key-value pairs. The data can be consumed
+in pods or provide the configurations for system components such as controllers. ConfigMap is
+similar to [Secrets](/docs/concepts/configuration/secret/), but provides a means of working
+with strings that don't contain sensitive information. Users and system components alike can
+store configuration data in ConfigMap.
 
 {{< note >}}
-ConfigMaps should reference properties files, not replace them. Think of the ConfigMap as representing something similar to the Linux `/etc` directory and its contents. For example, if you create a [Kubernetes Volume](/docs/concepts/storage/volumes/) from a ConfigMap, each data item in the ConfigMap is represented by an individual file in the volume.
+ConfigMaps should reference properties files, not replace them. Think of the ConfigMap as
+representing something similar to the Linux `/etc` directory and its contents. For example,
+if you create a [Kubernetes Volume](/docs/concepts/storage/volumes/) from a ConfigMap, each
+data item in the ConfigMap is represented by an individual file in the volume.
 {{< /note >}}
 
-The ConfigMap's `data` field contains the configuration data. As shown in the example below, this can be simple -- like individual properties defined using `--from-literal` -- or complex -- like configuration files or JSON blobs defined using `--from-file`.
+The ConfigMap's `data` field contains the configuration data. As shown in the example below,
+this can be simple -- like individual properties defined using `--from-literal` -- or complex --
+like configuration files or JSON blobs defined using `--from-file`.
```

Comment:
```suggestion
this can be simple (like individual properties defined using `--from-literal`) or complex
(like configuration files or JSON blobs defined using `--from-file`).
```
This is a little easier for my brain to parse, but I leave that to you whether or not you find it helpful.